Mastodon/Scene/Onboarding/ServerRules/MastodonServerRulesViewController.swift

```diff
@@ -13,22 +13,97 @@ import MetaTextKit
 import MastodonAsset
 import MastodonCore
 import MastodonLocalization
+import SwiftUI
 
-final class MastodonServerRulesViewController: UIViewController, NeedsDependency {
+struct MastodonServerRulesView: View {
```

Comment:
Why is a SwiftUI-rewrite necessary? What was the problem with the `UITableView`-implementation in combination with a `UINavigationController`?

Reply:
I don't see why this should be a UITableView and adopting the changes seemed much more straight forward using SwiftUI to me. I wouldn't want to write any new UIKit code if we can use SwiftUI.